ITMO_FS/utils/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .functions import *
 from .information_theory import *
 from .qpfs_body import qpfs_body
+from .DataChecker import DataChecker
```

Comment:
Are you sure we need to export this?

Reply:
the code does not compile if we do not export DataChecker (probably because it's located in a different folder?)